### ISSUE TITLE
Job Tags / Flags

### ISSUE THREAD

**Issue by kPOWz** [kPOWz/Deviner#93]
"it would be very helpful if we had the option to apply certain "flags" to jobs by way of highlighting an entire row with a given color.  The best example of this function would probably be highlighting a job in orange to show that it is waiting on backordered items before the job can be assigned to a printer for production.  I don't know if or how that fits in to your current surgical procedures, but it seemed wise to bring it up sooner rather than later." - @spacecowboy88

**Comment by benjanders** [comment]
I like the idea, I just haven't ever thought through the way that the flag is applied. I also have shied away from wanting to add too much functionality. The way it is currently working is that jobs just get stuck in the ordered or counted status. 

I don't think there would need to be a high priority on implementing the change, but just for documentation's sake, here are the areas where jobs get stuck:

-Backordered. Then it falls into two different categories: 
    -"Print it short." It's only short a few items and the customer is ok with it, so we will just print it.
    -"Wait until everything shows up." The customer isn't in a hurry, and wants a complete order.

-Misprinted Shirts. Then two categories:
    -"Send it short." Basically: The invoice needs to be changed so a customer isn't charged for the full order.
    -"Wait until a replacement comes in." Sometimes a customer won't come pick the job up until its complete.
    -"Print the replacement at a later date." If it takes a while to get a replacement, the customer will pick up the majority of the order then come back in a few weeks to get stragglers.

As far as I can tell, those are the main flags that would need to be added. It would be nice to have the additional information, but I can't figure out where to work it in. I like the idea of color coding the flags.

**Comment by kPOWz** [comment]
ok, really good info. 

yes, I didn't apply the 'GUS 1.0' milestone to this one just so we keep that queue/scope really clear for now.

From a Bootstrap design perspective, typically, this type of thing would be visually indicated with a Bootstrap label.  Using GUS 1.0 color story, the labels look like this : http://kpowz.github.io/gus-bootstrap/components/#labels
